Makefile

```diff
@@ -1,12 +1,12 @@
-DB_URL=postgres://postgres:12345@localhost:5432/postgres?sslmode=disable
+DB_URL=postgres://postgres:12345@db:5432/postgres?sslmode=disable
 
 .PHONY: migrate-up migrate-down create-migration proto
 
 proto:
 	protoc --proto_path=protobuf --go_out=proto --go_opt=paths=source_relative --go-grpc_out=proto --go-grpc_opt=paths=source_relative protobuf/*.proto
 
 run:
-	JWT_TOKEN_SECRET=my_secret_key REST_PORT=8080 GRPC_PORT=50052 ADDRESS=0.0.0.0 go run cmd/main.go
+	go run cmd/main.go
 
 lint:
 	 golangci-lint run --enable-all
```

Comment:
code is not linted, btw

Reply:
Also I would suggest more generalist approach to linting. E.g. guarantee version. You can change your make command as follows:

```
GOLANGCILINT ?= docker run --rm -v $(shell pwd):/app -w /app golangci/golangci-lint:v1.57.2 golangci-lint

.....
.PHONY: lint
lint:
	 $(GOLANGCILINT) run -v --enable-all
```

This way, even if user don't have lint installed, they can still run linter on the same version you used via docker.